pages/[contract]/[tokenId]/index.tsx

```diff
@@ -95,6 +95,7 @@ const Index: NextPage<Props> = ({ collectionId, tokenDetails }) => {
     ],
     includeTopBid: true,
     includeAttributes: true,
+    includeDynamicPricing: true,
   })
 
   const tokens = tokenData.data
```

Comment:
I think we should add `includeDynamicPricing: true` to getStaticProps here right?

Reply:
nice catch